glossary.md

```diff
@@ -0,0 +1,89 @@
+# Glossary of Optimistic Ethereum Terms
+
+### Layer 1 (L1)
+
+The Ethereum Mainnet.
```

Comment:
This technically could be a testnet. Perhaps a better definition could be an ethereum network that derives its security from proof of work/proof of stake

Reply:
```suggestion
An Ethereum network with its own native security mechanism (ie. proof of work, proof of stake or proof of authority).
```